calibration/CTDMOG/CGINS-CTDMOG-13426__20161014.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13426,CC_ptcb1,-1.750000e-004,
 37-13426,CC_ptcb2,0.00E+00,
 37-13426,CC_ptempa0,1.557536e+002,
-37-13426,CC_ptempa1,-6.889070e-002,
-37-13426,CC_ptempa2,5.792655e-007,
-37-13426,CC_wbotc,2.4329e-007,
+37-13426,CC_ptempa1,-6.889069e-02,
+37-13426,CC_ptempa2,5.792656e-07,
+37-13426,CC_wbotc,2.432850e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT